src/algebra/lie/nilpotent.lean

```diff
@@ -323,6 +329,71 @@ end
 
 end lie_module
 
+namespace lie_submodule
+
+variables {N₁ N₂ : lie_submodule R L M}
+
+/-- The upper (aka ascending) central series. -/
+def ucs (k : ℕ) : lie_submodule R L M → lie_submodule R L M :=
```

Comment:
Should this be called `upper_central_series` to match `lie_module.lower_central_series`?

Reply:
Oh never mind, I see we have `lie_submodule.lcs` already

Reply:
Can you add ``See also `lie_submodule.lcs`. `` here, and similarly in `lcs` referring to `ucs`?